roles/activemq/defaults/main.yml

```diff
@@ -9,4 +9,4 @@ activemq_environment:
     - -Xms128m
     - -Xmx1g
     - $ACTIVEMQ_OPTS
-activemq_version: 5.18.5
+activemq_version: null
```

Comment:
hat's the benefit vs removing it from here?

Reply:
the role in the current state is not really reusable (the download URL is not using this activemq_version) - so I prefer to get an explicit failure when we forget to pass this variable properly